app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] As per the given _ERD_, the _User_ **association** should have _author_id_ referenced as **foreign key** in _Post_, _Comment,_ and _Like_ models. Kindly ensure to improve this aspect, as can be seen below:
```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```